src/plugins/certauth/test/main.c

```diff
@@ -0,0 +1,239 @@
+/* -*- mode: c; c-basic-offset: 4; indent-tabs-mode: nil -*- */
+/* main.c - PKINIT certificate authorization plugin test modules.
```

Comment:
The comment formatting here does not exactly match what's in src/prototype/prototype.c.  That's actually important for some automated processing tools used by Oracle.